src/main/java/org/isf/utils/db/Auditable.java

```diff
@@ -72,7 +73,7 @@ public LocalDateTime getCreatedDate() {
         return TimeTools.truncateToSeconds(createdDate);
     }
 
-    public int getActive() {
+    public Integer getActive() {
```

Comment:
Personally I think that raw types (`int`) should be used in place of Objects as it just causes unnecessary boxing and unboxing of values......

Reply:
Agreed, thanks!